src/layouts/CollectionLayout.tsx

```diff
@@ -0,0 +1,183 @@
+import { CollectionRaisedCard } from '@/common/components/collections/CollectionRaisedCard';
+import {
+  CollectionTabRoute,
+  NFTCollection,
+} from '@/common/components/collections/collections.util';
+import { AvatarIcons } from '@/common/components/elements/Avatar';
+import { FollowUnfollowButton } from '@/common/components/elements/FollowUnfollowButton';
+import { User } from '@/common/components/feed/feed.utils';
+import { ProfileHandle, ProfilePFP } from '@/common/components/feed/FeedCard';
+import { HOLAPLEX_MARKETPLACE_SUBDOMAIN } from '@/common/constants/marketplace';
+import { useAnchorWallet, useConnection } from '@solana/wallet-adapter-react';
+import classNames from 'classnames';
+import Link from 'next/link';
+import { useRouter } from 'next/router';
+import { INITIAL_FETCH } from 'pages/profiles/[publicKey]/nfts';
+import React, { ReactElement, useMemo, useState } from 'react';
+import {
+  GetCollectionQuery,
+  useAllConnectionsToQuery,
+  useNftsInCollectionQuery,
+} from 'src/graphql/indexerTypes';
+
+function CreatorChip(props: { user: User }) {
+  return (
+    <span className="flex max-w-fit items-center space-x-4 py-3 px-2 text-base font-medium shadow-2xl">
+      <ProfilePFP user={props.user} />
+      <ProfileHandle user={props.user} />
+    </span>
+  );
+}
+
+export default function CollectionLayout({
+  children,
+  ...props
+}: {
+  collectionAddress: string;
```

Comment:
I think here you could use `: CollectionPage & {children: ReactElement}` instead of having to repeat the interface spec